src/components/commons/Header.tsx

```diff
@@ -47,6 +51,9 @@ export default function Header() {
   };
   const CartInfo = getSessionItem(SessionItems.cart) as Cart | undefined;
   const [drawstate, setDrawstate] = React.useState(false);
+  const [enableAuthentication, setEnableAuthentication] = React.useState(
+    getSessionItem(SessionItems.enableAuthentication) === "true"
+  );
   const ignoreRoutes: Array<string> = [
```

Comment:
```suggestion
  const ignoredRoutesForLogin: Array<string> = [
```
Or something like that, to make it clear that these are the paths that should not show the login button.